src/components/Comments.js

```diff
@@ -31,6 +31,23 @@ const toggleViewReplies = (viewRepliesToggled, setViewRepliesToggled, key) => {
       )
 }
 
+const intervals = [
+  { label: 'year', seconds: 31536000 },
+  { label: 'month', seconds: 2592000 },
+  { label: 'day', seconds: 86400 },
+  { label: 'hour', seconds: 3600 },
```

Comment:
@ayushsatyam146 let's use moment.js for this instead of implementing our own logic

Reply:
@kbhutani0001 as guided by you I have removed old logic and used moment.js now. Please review